### ISSUE TITLE
TensorRT 8.5 dynamic input shape: could not find any implementation for node {ForeignNode.... matmul....}

### ISSUE THREAD

**Issue by shuruiz** [NVIDIA/TensorRT#2599]
## Description

I encountered a bug similar to this one: https://github.com/NVIDIA/TensorRT/issues/2124 
Different from the above mentioned issue, I don't convert from onnx but using torch2trt to convert pytorch .pt model to trt engine.  I tried the latest tensorrt 8.5.2, and the problem still exits. The problem is mainly from the matmul. Is that only fixed for onnx model to trt engine case? Can you guys please fix the problem in general? My model has dynamic input as well.

In addition, once I removed the matmul the conversion worked fine. However the result had significant accuracy drop compared to torch model output. 

*With static input shape, the conversion worked fine. Previously I used trt 7.1, and that worked fine, no matter dynamic or static input shape.

## Environment

**TensorRT Version**: 8.5.2
**NVIDIA GPU**: T4
**NVIDIA Driver Version**: 450.119.04
**CUDA Version**: 10.2
**CUDNN Version**: 8.6.0
**Operating System**: Ubuntu 18.04
**Python Version (if applicable)**: 3.6
**Tensorflow Version (if applicable)**: 
**PyTorch Version (if applicable)**: 1.10
**Baremetal or Container (if so, version)**:

**Comment by oxana-nvidia** [comment]
Hi @shuruiz, could you please provide a repro: an example model and steps you are using to run it?
Is my understanding right, that with onnx model you can get the result that you are expecting? 
torch2trt is not maintained by TensorRT team. It can be a bug or not supported layer in torch2trt. Without more data provided it is hard to say.

**Comment by ttyio** [comment]
Closing since no activity for more than 3 weeks, please reopen if you still have question, thanks!
